paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/cdc/CdcMultiTableParsingProcessFunction.java

```diff
@@ -42,18 +42,22 @@
 public class CdcMultiTableParsingProcessFunction<T> extends ProcessFunction<T, Void> {
 
     private final EventParser.Factory<T> parserFactory;
+    private final boolean caseSensitive;
 
     private transient EventParser<T> parser;
     private transient Map<String, OutputTag<List<DataField>>> updatedDataFieldsOutputTags;
     private transient Map<String, OutputTag<CdcRecord>> recordOutputTags;
 
-    public CdcMultiTableParsingProcessFunction(EventParser.Factory<T> parserFactory) {
+    public CdcMultiTableParsingProcessFunction(
+            EventParser.Factory<T> parserFactory, boolean caseSensitive) {
         this.parserFactory = parserFactory;
+        this.caseSensitive = caseSensitive;
```

Comment:
It is OK to just let `EventParser.Factory` know the `caseSensitive` in actions.
We don't need to modify `CdcMultiTableParsingProcessFunction`.